test262/main.css

```diff
@@ -115,3 +115,59 @@ code {
   border-radius: 10px;
   padding: 20px;
 }
+
+.hidden {
+  display: none;
+}
+
+.tree-node-status-container {
```

Comment:
Can we move all of these to a new CSS file? `per-file/main.css` or something like that.